docs/proposal-optional-output-extension.md

```diff
@@ -114,40 +133,36 @@ There are currently three problems with expiry (as of cylc-flow 8.1.4):
    }
    ```
 
-   Note: the completion condition only covers outputs generated when a task is executed - it cannot
-   include `:expired`, `:submit` or `:submit-fail`. It also cannot include `started` (this has to
-   happen if a task executes) or `finished` (this isn't an output).
```

Comment:
This restriction is no longer necessary with the above change.

We could decide to preserve this restriction, however, it will be clearer for users if we broaden the scope of the completion condition to include pre-conditions as otherwise we will have to automatically add the pre-condition logic internally.